src/Button/DigitizeButton/DigitizeButton.jsx

```diff
@@ -897,7 +907,8 @@ class DigitizeButton extends React.Component {
     if (feature.get('isLabel')) {
       this._digitizeTextFeature = feature;
       this.setState({
-        showLabelPrompt: true
+        showLabelPrompt: true,
+        textLabel: feature.getStyle().getText().getText() || ''
```

Comment:
Can `feature.getStyle().getText()` become undefined here?

Reply:
No, since we check above if feature has property `isLabel`, which is extra set on labeled features only